### PR TITLE
Package refresh and some fixes

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
- Upgrade dependencies (ava 3 and eslint 7 included) :outbox_tray:
- Fix `_waitFor` unknown issue reported in #145 for `serverless-offline-kinesis` and `serverless-offline-dynamodb-streams` :bug:
- Fix broken sqs arn reported by #133 `serverless-offline-sqs` :label:
- Adjustements and stabilisation of the integration tests :balance_scale:

For more details, see commit list.

### CODE REVIEW

**Review by godu — APPROVED** [review]

File: packages/dynamodb-streams-readable/test/index.js [hunk 1]
```diff
@@ -4,35 +4,34 @@ const DynamoDB = require('aws-sdk/clients/dynamodb');
...
-  );
+  dynamodb
+    .batchWriteItem({
+      RequestItems: {
+        [tableName]: items.map(document => ({
+          PutRequest: document
+        }))
+      }
+    })
+    .promise();
```
Comment (Fausto95) [inline packages/dynamodb-streams-readable/test/index.js:21]: ✅

File: packages/dynamodb-streams-readable/test/index.js [hunk 2]
```diff
@@ -209,30 +203,30 @@ test.serial('reads latest records', async t => {
...
-        .on('error', function(err) {
+        .on('error', function (err) {
           t.fail('should not error');
           reject(err);
-        })
-    ),
-    wait(100).then(() => batchWriteItem(dynamodb, tableName, subsequentDocuments))
+        });
+    }),
+    delay(100).then(() => batchWriteItem(dynamodb, tableName, subsequentDocuments))
```
Comment (Fausto95) [inline packages/dynamodb-streams-readable/test/index.js:229]: next change will be `defer` ? 😆 just kidding

File: tests/serverless-plugins-integration/test-dynamodb-streams.js [hunk 3]
```diff
@@ -11,63 +12,37 @@ const client = new DynamoDB({
...
-      .putItem({
-        Item: {id: {S: `Bug`}},
-        TableName: 'MyFourthTable'
-      })
-      .promise()
-  ]);
-  // wait stream get a new iterator
-  await new Promise(resolve => setTimeout(resolve, 1000));
+// Dynamodb-local doesn't create stream until table isn't empty
+const unemptyTables = () =>
```
Comment (Fausto95) [inline tests/serverless-plugins-integration/test-dynamodb-streams.js:16]: ✅

**Review by Fausto95 — APPROVED** [review]

File: packages/dynamodb-streams-readable/test/index.js (hunk 2 above)
Comment (AdrieanKhisbe) [reply packages/dynamodb-streams-readable/test/index.js:229]: `deferBecauseFaustPrefersIt` :trollface:

**Comment by AdrieanKhisbe** [comment]
Successfully published:
 - dynamodb-streams-readable@1.1.0
 - serverless-apigateway-access-logs@0.2.6
 - serverless-offline-dynamodb-streams@4.2.0
 - serverless-offline-kinesis@4.2.0
 - serverless-offline-s3@4.2.0
 - serverless-offline-sqs@4.1.1
 - serverless-offline-ssm-provider@2.1.0

**Pull request merged**